server/views/partials/new_workAndSkillsPage/goals/noGoals.njk

```diff
@@ -0,0 +1,5 @@
+{# Goals summary card content for when the prisoner has no goals #}
+
+<p class="govuk-!-margin-bottom-7">The careers information, advice and guidance (CIAG) team set these goals. They do not include sentence plan targets.</p>
+<p class="govuk-body">{{ workAndSkillsPrisonerName }} has no goals recorded yet.</p>
+<p class="govuk-body"><a href="{{ config.serviceUrls.learningAndWorkProgress }}/plan/{{ prisonerNumber }}/view/overview">Add goals in the Learning and work progress service</a></p>
```

Comment:
The link to the PLP UI needs to be conditional on the user having the correct role
They need to have the role `ROLE_EDUCATION_WORK_PLAN_EDITOR`
If the user does not have this role then we should not display this link (same comment on all the pages where we have a link to the PLP UI service)